Makefile.am

```diff
@@ -1748,15 +1748,15 @@ snapshot::
 	tar -C snapshot -c -f - $(PACKAGE_NAME)-$(GIT_VERSION) | gzip -9 > $(PACKAGE_NAME)-$(GIT_VERSION).tar.gz
 	@rm -rf snapshot
 
-docsrc/imap/admin/systemcommands/cyradm.rst: $(top_srcdir)/perl/imap/IMAP/Shell.pm
+docsrc/imap/reference/manpages/systemcommands/cyradm.rst: $(top_srcdir)/perl/imap/IMAP/Shell.pm
```

Comment:
These three filenames also need to be changed a bit further down in the "GENERATED_RSTS" variable, around line 1787

Also, around line 55 of .gitignore